src/cljs/proton/lib/helpers.cljs

```diff
@@ -68,7 +69,7 @@
         is-category? ((comp not nil?) category)
         class-name (if is-category? "proton-key-category" "proton-key-action")
         value (if is-category? (str "+" category) (or title action))
-        keystroke (name (key keybinding))]
+        keystroke (-> keybinding first str rest join)]
```

Comment:
This change was necessary because the (name (keyword "/")) somehow always returned an empty string meaning the keybinding was empty (probably due to escaping).

Now I'm converting the keybinding to a string, strip the first character and join it together. clojure.string is then taking care of escaping for us.